### PR TITLE
fix(suite): change key for translation variable

### PR DESCRIPTION
**Pull request by komret**
## Description

Fix key mismatch in translation.

## Screenshots:
image

### CODE REVIEW

File: packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/PayoutCard.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const PayoutCard = ({
                             values={{ days: BACKUP_REWARD_PAYOUT_DAYS }}
                         />
                     ) : (
-                        <Translation id="TR_STAKE_DAYS" values={{ count: payout }} />
+                        <Translation id="TR_STAKE_DAYS" values={{ days: payout }} />
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/PayoutCard.tsx:51]:
```suggestion
                        <Translation id="TR_STAKE_DAYS" values={{ count: payout }} />
```

**Review by tomasklim — CHANGES_REQUESTED** [review]
```
    TR_STAKE_DAYS: {
        id: 'TR_STAKE_DAYS',
        defaultMessage: '{count, plural, one {# day} few {# days} other {# days}}'
},
```

https://github.com/trezor/trezor-suite/pull/12742/files#r1631042071

File: packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/PayoutCard.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/PayoutCard.tsx:51]: Moreover, `TR_STAKE_DAYS` is used on 3 places

**Review by peter-sanderson — APPROVED** [review]

**Comment by komret** [comment]
Should be resolved by the next Crowdin sync, closing.

**Pull request closed without merging**